src/util/offscreen_canvas_supported.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import window from './window';
+
+let supportsOffscreenCanvas = null;
+
+export default function offscreenCanvasSupported(): boolean {
+    if (supportsOffscreenCanvas == null) {
```

Comment:
nit: `===`

Reply:
Intentional as a `null/undefined` check

Reply:
Then let's use

```suggestion
    if (!supportsOffscreenCanvas) {
```

Reply:
But then that would run when `supportsOffscreenCanvas` is `false` as well.

Reply:
In what situations would `supportsOffscreenCanvas` be `undefined` then? Let's add some flowtype to the variable to make it clearer what states it can be in.

Reply:
It really wound't ever be `undefined` I was just sticking to the most easiest `maybe type` refinement implementation for flow( I guess i still forgot the annotation) https://flow.org/en/docs/types/maybe/#toc-refining-maybe-types (the 2nd example)

I added the `?boolean` annotation.